src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -4985,7 +4985,7 @@ Unify Enum declarations with other block type declarations.
         <native-equiv name="Expression.Opcodes.Nothing" />
       </node-kind>
 
-      <child name="Token" kind="IntegerLiteralToken|CharacterLiteralToken|DecimalLiteralToken|FloatingLiteralToken|DateLiteralToken|StringLiteralToken|TrueKeyword|FalseKeyword|NothingKeyword">
+      <child name="Token" kind="IntegerLiteralToken|CharacterLiteralToken|DecimalLiteralToken|FloatingLiteralToken|DateLiteralToken|StringLiteralToken|DollarSignDoubleQuoteToken|DoubleQuoteToken|InterpolatedStringTextToken|TrueKeyword|FalseKeyword|NothingKeyword">
```

Comment:
This change doesn't feel appropriate to me. If a token cannot be a valid child of a LiteralExpressionSyntax, its kind should not be listed here. The purpose of SyntaxFacts.IsLiteralExpressionToken is to check whether a token is a valid child of a LiteralExpressionSyntax, it is used this way in many places within compiler. It sounds like IDE code should have its own helper that would make use of SyntaxFacts.IsLiteralExpressionToken and will handle additional tokens given the particular need/situation.